diff --git a/node_modules/@tanstack/virtual-core/build/lib/index.d.ts b/node_modules/@tanstack/virtual-core/build/lib/index.d.ts
index 3eb35ec..618e287 100644
--- a/node_modules/@tanstack/virtual-core/build/lib/index.d.ts
+++ b/node_modules/@tanstack/virtual-core/build/lib/index.d.ts
@@ -21,7 +21,7 @@ export interface VirtualItem {
     end: number;
     size: number;
 }
-interface Rect {
+export interface Rect {
     width: number;
     height: number;
 }
