diff --git a/node_modules/native-keymap/binding.gyp b/node_modules/native-keymap/binding.gyp
index a2a3876..438b276 100644
--- a/node_modules/native-keymap/binding.gyp
+++ b/node_modules/native-keymap/binding.gyp
@@ -1,4 +1,7 @@
 {
+  'variables' : {
+    'openssl_fips': '',
+  }
   "targets": [
     {
       "target_name": "keymapping",
diff --git a/node_modules/native-keymap/build/Release/keymapping.iobj b/node_modules/native-keymap/build/Release/keymapping.iobj
index 7b810d7..3afb978 100644
Binary files a/node_modules/native-keymap/build/Release/keymapping.iobj and b/node_modules/native-keymap/build/Release/keymapping.iobj differ
diff --git a/node_modules/native-keymap/build/Release/keymapping.ipdb b/node_modules/native-keymap/build/Release/keymapping.ipdb
index b634398..eb39f1d 100644
Binary files a/node_modules/native-keymap/build/Release/keymapping.ipdb and b/node_modules/native-keymap/build/Release/keymapping.ipdb differ
diff --git a/node_modules/native-keymap/build/Release/keymapping.node b/node_modules/native-keymap/build/Release/keymapping.node
index 6ea5668..f1c5af1 100644
Binary files a/node_modules/native-keymap/build/Release/keymapping.node and b/node_modules/native-keymap/build/Release/keymapping.node differ
diff --git a/node_modules/native-keymap/build/Release/keymapping.pdb b/node_modules/native-keymap/build/Release/keymapping.pdb
index 6415784..77d00fa 100644
Binary files a/node_modules/native-keymap/build/Release/keymapping.pdb and b/node_modules/native-keymap/build/Release/keymapping.pdb differ
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.node.recipe b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.node.recipe
index d617274..86ae92b 100644
--- a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.node.recipe
+++ b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.node.recipe
@@ -2,7 +2,7 @@
 <Project>
   <ProjectOutputs>
     <ProjectOutput>
-      <FullPath>C:\Users\Wuriko\AppData\Local\Temp\tmp-12604Rv5uyLS6Khv0\node_modules\native-keymap\build\Release\keymapping.node</FullPath>
+      <FullPath>C:\Users\Wuriko\Desktop\dev\nhs\izabela\node_modules\native-keymap\build\Release\keymapping.node</FullPath>
     </ProjectOutput>
   </ProjectOutputs>
   <ContentFiles />
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/CL.command.1.tlog b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/CL.command.1.tlog
index e18c53b..13950d0 100644
Binary files a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/CL.command.1.tlog and b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/CL.command.1.tlog differ
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/CL.read.1.tlog b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/CL.read.1.tlog
index 1d01035..2cef39d 100644
Binary files a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/CL.read.1.tlog and b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/CL.read.1.tlog differ
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/CL.write.1.tlog b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/CL.write.1.tlog
index f098e69..0e0c026 100644
Binary files a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/CL.write.1.tlog and b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/CL.write.1.tlog differ
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/keymapping.lastbuildstate b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/keymapping.lastbuildstate
index 39b2f69..aae4980 100644
--- a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/keymapping.lastbuildstate
+++ b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/keymapping.lastbuildstate
@@ -1,2 +1,2 @@
 PlatformToolSet=v142:VCToolArchitecture=Native64Bit:VCToolsVersion=14.29.30133:TargetPlatformVersion=10.0.19041.0:
-Release|x64|C:\Users\Wuriko\AppData\Local\Temp\tmp-12604Rv5uyLS6Khv0\node_modules\native-keymap\build\|
+Release|x64|C:\Users\Wuriko\Desktop\dev\nhs\izabela\node_modules\native-keymap\build\|
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/keymapping.write.1u.tlog b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/keymapping.write.1u.tlog
index fb37a0c..2c09286 100644
Binary files a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/keymapping.write.1u.tlog and b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/keymapping.write.1u.tlog differ
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/link.command.1.tlog b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/link.command.1.tlog
index 027ab44..8c9c6a0 100644
Binary files a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/link.command.1.tlog and b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/link.command.1.tlog differ
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/link.read.1.tlog b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/link.read.1.tlog
index 852f72f..8621613 100644
Binary files a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/link.read.1.tlog and b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/link.read.1.tlog differ
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/link.write.1.tlog b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/link.write.1.tlog
index 8b240a8..99f7b24 100644
Binary files a/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/link.write.1.tlog and b/node_modules/native-keymap/build/Release/obj/keymapping/keymapping.tlog/link.write.1.tlog differ
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/src/keyboard_win.obj b/node_modules/native-keymap/build/Release/obj/keymapping/src/keyboard_win.obj
index 93236f2..7a22c82 100644
Binary files a/node_modules/native-keymap/build/Release/obj/keymapping/src/keyboard_win.obj and b/node_modules/native-keymap/build/Release/obj/keymapping/src/keyboard_win.obj differ
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/src/keymapping.obj b/node_modules/native-keymap/build/Release/obj/keymapping/src/keymapping.obj
index 05bc2bc..96fb64d 100644
Binary files a/node_modules/native-keymap/build/Release/obj/keymapping/src/keymapping.obj and b/node_modules/native-keymap/build/Release/obj/keymapping/src/keymapping.obj differ
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/src/string_conversion.obj b/node_modules/native-keymap/build/Release/obj/keymapping/src/string_conversion.obj
index 7017636..d93ab48 100644
Binary files a/node_modules/native-keymap/build/Release/obj/keymapping/src/string_conversion.obj and b/node_modules/native-keymap/build/Release/obj/keymapping/src/string_conversion.obj differ
diff --git a/node_modules/native-keymap/build/Release/obj/keymapping/win_delay_load_hook.obj b/node_modules/native-keymap/build/Release/obj/keymapping/win_delay_load_hook.obj
index 991b5c1..084091b 100644
Binary files a/node_modules/native-keymap/build/Release/obj/keymapping/win_delay_load_hook.obj and b/node_modules/native-keymap/build/Release/obj/keymapping/win_delay_load_hook.obj differ
diff --git a/node_modules/native-keymap/build/binding.sln b/node_modules/native-keymap/build/binding.sln
index 0d4675f..780ca3d 100644
--- a/node_modules/native-keymap/build/binding.sln
+++ b/node_modules/native-keymap/build/binding.sln
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 2015
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "keymapping", "keymapping.vcxproj", "{72E045B4-615B-62CB-6341-03463EFADABC}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "keymapping", "keymapping.vcxproj", "{6C15F37D-C094-705B-0477-74FFD8FF0ED8}"
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
@@ -8,10 +8,10 @@ Global
 		Release|x64 = Release|x64
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{72E045B4-615B-62CB-6341-03463EFADABC}.Debug|x64.ActiveCfg = Debug|x64
-		{72E045B4-615B-62CB-6341-03463EFADABC}.Debug|x64.Build.0 = Debug|x64
-		{72E045B4-615B-62CB-6341-03463EFADABC}.Release|x64.ActiveCfg = Release|x64
-		{72E045B4-615B-62CB-6341-03463EFADABC}.Release|x64.Build.0 = Release|x64
+		{6C15F37D-C094-705B-0477-74FFD8FF0ED8}.Debug|x64.ActiveCfg = Debug|x64
+		{6C15F37D-C094-705B-0477-74FFD8FF0ED8}.Debug|x64.Build.0 = Debug|x64
+		{6C15F37D-C094-705B-0477-74FFD8FF0ED8}.Release|x64.ActiveCfg = Release|x64
+		{6C15F37D-C094-705B-0477-74FFD8FF0ED8}.Release|x64.Build.0 = Release|x64
 	EndGlobalSection
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
diff --git a/node_modules/native-keymap/build/keymapping.vcxproj b/node_modules/native-keymap/build/keymapping.vcxproj
index 6095c2f..0fc424a 100644
--- a/node_modules/native-keymap/build/keymapping.vcxproj
+++ b/node_modules/native-keymap/build/keymapping.vcxproj
@@ -11,7 +11,7 @@
     </ProjectConfiguration>
   </ItemGroup>
   <PropertyGroup Label="Globals">
-    <ProjectGuid>{72E045B4-615B-62CB-6341-03463EFADABC}</ProjectGuid>
+    <ProjectGuid>{6C15F37D-C094-705B-0477-74FFD8FF0ED8}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>keymapping</RootNamespace>
     <IgnoreWarnCompileDuplicatedFilename>true</IgnoreWarnCompileDuplicatedFilename>
@@ -146,7 +146,7 @@
     <ClCompile Include="..\src\keyboard_win.cc">
       <ObjectFileName>$(IntDir)\src\keyboard_win.obj</ObjectFileName>
     </ClCompile>
-    <ClCompile Include="C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\src\win_delay_load_hook.cc"/>
+    <ClCompile Include="C:\Users\Wuriko\Desktop\dev\nhs\izabela\node_modules\node-gyp\src\win_delay_load_hook.cc"/>
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets"/>
   <Import Project="$(VCTargetsPath)\BuildCustomizations\masm.targets"/>
diff --git a/node_modules/native-keymap/build/keymapping.vcxproj.filters b/node_modules/native-keymap/build/keymapping.vcxproj.filters
index 5928704..1b399e0 100644
--- a/node_modules/native-keymap/build/keymapping.vcxproj.filters
+++ b/node_modules/native-keymap/build/keymapping.vcxproj.filters
@@ -22,25 +22,31 @@
     <Filter Include="C:">
       <UniqueIdentifier>{7B735499-E5DD-1C2B-6C26-70023832A1CF}</UniqueIdentifier>
     </Filter>
-    <Filter Include="C:\Program Files">
-      <UniqueIdentifier>{92EF4BA8-6BC2-65D1-451F-28EBD4AE726A}</UniqueIdentifier>
+    <Filter Include="C:\Users">
+      <UniqueIdentifier>{E9F714C1-DA89-54E2-60CF-39FEB20BF756}</UniqueIdentifier>
     </Filter>
-    <Filter Include="C:\Program Files\nodejs">
-      <UniqueIdentifier>{A3C8E949-BCF6-0C67-6656-340A2A097708}</UniqueIdentifier>
+    <Filter Include="C:\Users\Wuriko">
+      <UniqueIdentifier>{415EA44A-D4D4-1720-38D2-EA54332F5553}</UniqueIdentifier>
     </Filter>
-    <Filter Include="C:\Program Files\nodejs\node_modules">
-      <UniqueIdentifier>{56DF7A98-063D-FB9D-485C-089023B4C16A}</UniqueIdentifier>
+    <Filter Include="C:\Users\Wuriko\Desktop">
+      <UniqueIdentifier>{B23D6E4D-7634-EF57-62FC-46A6189C0B6E}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="C:\Users\Wuriko\Desktop\dev">
+      <UniqueIdentifier>{51D92A3A-C0F8-78D6-DFF5-4F152A3AB774}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="C:\Users\Wuriko\Desktop\dev\nhs">
+      <UniqueIdentifier>{95EF1E90-6993-695E-CF21-0EF3CB757B1F}</UniqueIdentifier>
     </Filter>
-    <Filter Include="C:\Program Files\nodejs\node_modules\npm">
-      <UniqueIdentifier>{741E0E76-39B2-B1AB-9FA1-F1A20B16F295}</UniqueIdentifier>
+    <Filter Include="C:\Users\Wuriko\Desktop\dev\nhs\izabela">
+      <UniqueIdentifier>{F84F5E53-22AD-7873-318A-D2DA3EEA47DF}</UniqueIdentifier>
     </Filter>
-    <Filter Include="C:\Program Files\nodejs\node_modules\npm\node_modules">
+    <Filter Include="C:\Users\Wuriko\Desktop\dev\nhs\izabela\node_modules">
       <UniqueIdentifier>{56DF7A98-063D-FB9D-485C-089023B4C16A}</UniqueIdentifier>
     </Filter>
-    <Filter Include="C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp">
+    <Filter Include="C:\Users\Wuriko\Desktop\dev\nhs\izabela\node_modules\node-gyp">
       <UniqueIdentifier>{77348C0E-2034-7791-74D5-63C077DF5A3B}</UniqueIdentifier>
     </Filter>
-    <Filter Include="C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\src">
+    <Filter Include="C:\Users\Wuriko\Desktop\dev\nhs\izabela\node_modules\node-gyp\src">
       <UniqueIdentifier>{8CDEE807-BC53-E450-C8B8-4DEBB66742D4}</UniqueIdentifier>
     </Filter>
     <Filter Include="..">
@@ -57,8 +63,8 @@
     <ClCompile Include="..\src\keyboard_win.cc">
       <Filter>..\src</Filter>
     </ClCompile>
-    <ClCompile Include="C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\src\win_delay_load_hook.cc">
-      <Filter>C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\src</Filter>
+    <ClCompile Include="C:\Users\Wuriko\Desktop\dev\nhs\izabela\node_modules\node-gyp\src\win_delay_load_hook.cc">
+      <Filter>C:\Users\Wuriko\Desktop\dev\nhs\izabela\node_modules\node-gyp\src</Filter>
     </ClCompile>
     <None Include="..\binding.gyp">
       <Filter>..</Filter>
